src/test/java/org/opensearch/security/authtoken/jwt/EncryptionDecryptionUtilTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.authtoken.jwt;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.Base64;
+
+public class EncryptionDecryptionUtilTest {
+
+    @Test
+    public void testEncryptDecrypt() {
+        String secret = Base64.getEncoder().encodeToString("mySecretKey12345".getBytes());
+        String data = "Hello, OpenSearch!";
+
+        String encryptedString = EncryptionDecryptionUtil.encrypt(secret, data);
+        String decryptedString = EncryptionDecryptionUtil.decrypt(secret, encryptedString);
+
+        Assert.assertEquals(data, decryptedString);
+    }
+
+    @Test
+    public void testDecryptingWithWrongKey() {
+        String secret1 = Base64.getEncoder().encodeToString("correctKey12345".getBytes());
+        String secret2 = Base64.getEncoder().encodeToString("wrongKey1234567".getBytes());
+        String data = "Hello, OpenSearch!";
+
+        String encryptedString = EncryptionDecryptionUtil.encrypt(secret1, data);
+
+        try {
+            EncryptionDecryptionUtil.decrypt(secret2, encryptedString);
+            Assert.fail("Should have thrown an exception when decrypting with a wrong key");
+        } catch (RuntimeException e) {
+            // Expected exception
```

Comment:
We should check for exception message here. Also you can re-write this as:
```
RuntimeException ex = assertThrows(
            RuntimeException.class,
            () -> EncryptionDecryptionUtil.decrypt(secret2, encryptedString);
        );
assertEquals("action [actionName] already registered", ex.getMessage());
```

Reply:
Fixed~